FilmFestivalLoader/IFFR/parse_iffr_html.py

```diff
@@ -531,63 +484,66 @@ class FilmInfoParseState(Enum):
         IN_ARTICLE = auto()
         IN_PARAGRAPH = auto()
         IN_EMPHASIS = auto()
+        IN_SCREENED_FILM_LIST = auto()
         AWAITING_SCREENINGS = auto()
-        AWAITING_COMBINATION_LINK = auto()
-        IN_COMBINATION_LINK = auto()
+        AWAITING_SCREENED_FILM_LINK = auto()
+        IN_SCREENED_FILM_LINK = auto()
         DONE = auto()
 
     debugging = DEBUGGING
-    intro_span = datetime.timedelta(minutes=4)
-    combination_loaded_by_url = {}
 
-    def __init__(self, festival_data, film, combi_keeper, charset):
-        ScreeningParser.__init__(self, festival_data, combi_keeper, debug_recorder, 'FI', self.debugging)
+    def __init__(self, festival_data, film, charset):
+        ScreeningParser.__init__(self, festival_data, 'FI', self.debugging)
         self.festival_data = festival_data
         self.film = film
-        self.charset =  charset
+        self.charset = charset
+        self.event_is_combi = film.title in COMBINATION_EVENT_TITLES and has_category(film, Film.category_events)
         self.article_paragraphs = []
         self.article_paragraph = ''
         self.article = None
-        self.combination_slug = None
+        self.screened_film_slugs = []
 
         # Print a bar in the debug file when debugging.
         self.print_debug(self.bar, f'Analysing FILM {self.film} {self.film.url}')
 
         # Initialize the state stack.
         self.state_stack = self.StateStack(self.print_debug, self.FilmInfoParseState.IDLE)
 
-        # Get the film info of the current film. Its unique existence is guaranteed in AzPageParser.
+        # Get the film info of the current film. Its unique existence is guaranteed in AzPageParser. NOT YET!
```

Comment:
The team allows you to keep this warning.
Until the festival programma is published and loadable.